cluster/addons/dns/kube2sky/Makefile

```diff
@@ -1,13 +1,21 @@
-all: kube2sky
+# Makefile for the Docker image gcr.io/google_containers/kube2sky
+# MAINTAINER: thockin@google.com
+# If you update this image please bump the tag value before pushing.
+
+.PHONY: all kube2sky container push clean
+
+TAG = 1.4
+
+all: kube2sky container push
 
 kube2sky: kube2sky.go
 	CGO_ENABLED=0 godep go build -a -installsuffix cgo --ldflags '-w' ./kube2sky.go
 
 container: kube2sky
-	docker build -t gcr.io/google_containers/kube2sky .
+	docker build -t gcr.io/google_containers/kube2sky:$(TAG) .
```

Comment:
maybe we should parameterize the prefix, too?  When I was testing I would push to thockin/kube2sky and run that.